respy/simulate.py

```diff
@@ -278,36 +278,45 @@ def _simulate_single_period(df, state_space, optim_paras):
     columns = create_state_space_columns(optim_paras)
     indices = state_space.indexer[period][tuple(df[col].astype(int) for col in columns)]
 
-    # Get continuation values. Indices work on the complete state space whereas
-    # continuation values are period-specific. Make them period-specific.
-    continuation_values = state_space.get_continuation_values(period)
-    cont_indices = indices - state_space.slices_by_periods[period].start
+    try:
+        wages = state_space.wages[indices]
+        nonpecs = state_space.nonpec[indices]
+        # Get continuation values. Indices work on the complete state space whereas
+        # continuation values are period-specific. Make them period-specific.
+        cont_indices = indices - state_space.slices_by_periods[period].start
+        continuation_values = state_space.get_continuation_values(period)[cont_indices]
+
+        is_inadmissible = state_space.is_inadmissible[indices]
+    except IndexError as e:
```

Comment:
I think that is definitely helpful. Thank you for the fast implementation. 
Do you think it might make sense to also raise an error or warning in the model processing section if one specifies a model without filters that however puts positive probability on each lagged choice or do you think that would be too specific? (My hunch is that it would be too specific)

Reply:
Not sure if I am only lazy, but I would wait 😄. Maybe we can automatically generate some of the filters and then the warning would be redundant again.